README.adoc

```diff
@@ -112,34 +112,41 @@ include::finish/src/main/java/io/openliberty/guides/consumingrest/model/Artist.j
 include::finish/src/main/java/io/openliberty/guides/consumingrest/model/Album.java[tags=!comment]
 ----
 
-=== Using Jackson
-
-In the case of using Jackson for JSON binding, a few restrictions are applicable to these objects,
-they:
+=== Using JSON-B
 
-* Cannot have a constructor unless it is empty.
-* Must contain the same number of instance members as there are keys in the target JSON.
+JSON-B is a new feature introduced with Java EE 8 and strengthens Java's support for JSON.
+This guide previously used Jackson for JSON binding.  JSON-B requires, by default, a
+public default no-arguments constructor for deserialization.
 
-While Jackson annotations are not always necessary, they are considered good practice when used:
+The JSON-B engine includes a set of default mapping rules which can be run without
+any customization annotations or custom configuration. There are instances where you may
+find it useful to deserialize a JSON message with only certain properties, specific
+property names, or have a class with a custom constructor. In these cases,
+annotations are necessary and recommended:
 
-* The `@JsonProperty` annotation is used to map JSON keys to class instance members and vice versa.
-Without the use of this annotation, Jackson will attempt to do POJO mapping, matching the keys in
+* The `@JsonbProperty` annotation is used to map JSON keys to class instance members and vice versa.
+Without the use of this annotation, JSON-B will attempt to do POJO mapping, matching the keys in
 the JSON to the class instance members by name. If these names do not match, then getters and setters
 are matched to JSON keys and the instance members are set explicitly. If the getter and setter
 signatures do not match any key names, then deserialization fails. Note that the `Artist` POJO,
 does not require this annotation since all instance members match the JSON keys by name.
 
-* The `@JsonIgnore` annotation is used to define an object property that does not map to any JSON
+* The `@JsonbCreator` and `@JsonbProperty` annotations are used to annotate a custom constructor.
+This is required for proper parameter substitution when a custom constructor is used.
```

Comment:
- What about, "You can use the `@JsonbCreator` and `@JsonbProperty` annotations to annotate a custom constructor." These edits remove passive voice and change it to active voice.
- Avoid using "this" as a standalone word but instead say, "this _thing_." "This" by itself can be ambiguous. Maybe say, "The annotations are required for proper...'